tsconfig.json

```diff
@@ -24,6 +24,15 @@
     "noFallthroughCasesInSwitch": true,
     "noImplicitOverride": true,
     "skipLibCheck": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "paths": {
+      "@interfaces": ["src/interfaces"],
```

Comment:
Find a way to build this properly into JS first before using them here

Reply:
??

Reply:
Most of the settings don't properly compile to JS. If you've run the code using any of the scripts that build it, then fine. If not, then check if there's any error, and if yes, fix it.

Reply:
Yeah. For me it compiles and works.